GVFS/GVFS.Common/Paths.Shared.cs

```diff
@@ -13,10 +13,24 @@ public static string GetGitEnlistmentRoot(string directory)
 
         public static string GetServiceDataRoot(string serviceName)
         {
-            return Path.Combine(
+            if (Environment.OSVersion.Platform == PlatformID.Win32NT)

```

Comment:
`if (RuntimeInformation.IsOSPlatform(OSPlatform.Windows))`

Reply:
As discussed in an earlier comment, can we just move `GetServiceDataRoot` to `GVFSPlatform`?

If it's needed by GVFS.Hooks it can be added to `MacPlatform.Shared.cs` and `WindowsPlatform.Shared.cs`.